README.md

```diff
@@ -0,0 +1,334 @@
+to generate it, run:
+
+```shell
+openssl ec -in ec-ed25519-priv-key.pem -pubout > ec-ed25519-pub-key.pem
+```
+
+The private key, for Plone, needs to be within reach of the Plone instance,
+while the public key, for Camunda, needs to be directly in this repository,
+otherwise `docker-compose.yml` will not find it.
+
+#### Camunda cockpit
+
+The Camunda cockpit, its web UI to manage processes, can be found at:
+
+<http://localhost:8081/camunda/app/cockpit/default/#/login>
+
+Use `admin` for both user and password.
+
+You should see a three pane layout with the middle column having entries of all your uploaded processes.
+
+If you still haven't uploaded any, [learn how to do it](#publish-processes).
+
+### Plone integration
+
+With [processes created](#processes-modeler) and [Camunda running](#camunda-platform-7)
+its time for the last piece of the puzzle, and _hopefully the easiest_.
+
+Install `collective.bpmproxy` by adding it to your buildout
+and set two environment variables for the Camunda URL and the [private key](#authentication-keys):
+
+__Note:__ adjust accordingly if you are not using `zc.buildout`.
+
+```ini
+[buildout]
+...
+[instance]
+eggs =
+    ...
+    collective.bpmproxy
+environment-vars =
+    ...
+    CAMUNDA_API_URL http://localhost:8081/engine-rest
+    CAMUNDA_API_PRIVATE_KEY ${buildout:bin-directory}/../ec-ed25519-priv-key.pem
+```
+
+Run buildout and start Plone:
+
+```shell
+buildout
+./bin/instance fg
+```
+
+Create a Plone (Classic UI) instance and install the `collective.bpmproxy` add-on.
+
+There are two main ways to integrate processes with Plone:
+
+- via a content type: then the process _needs_ a form as a first step
+- via a signal: a Plone content rule needs to be installed globally or on a folder/document.
+  `collective.bpmproxy` already provides a few content rules ready to be used
+
+#### Content type integration
+
+On the Plone Toolbar's `Add` menu you should see a new content type: `Bpm Proxy`.
+
+This new content type allows to select (with autocompletion!) a process you want to manage.
+
+After creating the document, fill it and monitor the process in [Camunda's Cockpit](#camunda-cockpit).
+
+#### Content rules integration
+
+For processes that should happen automatically _given a certain condition_,
+install a `collective.bpmproxy` provided content rule with the following steps:
+
+- go to the folder where the content rule should be applied
+- on the Plone toolbar select the `Rules` option
+- on the view that opens, select from the drop down the content rule that suits your needs
+- configure the installed content rule as any other content rules
+
+If, for example, you selected the rule `BPMN signal: submitted any`
+your BPMN process needs to have a `Signal StartEvent` element
+with a `Signal` configured to `plone-content-submitted`.
+
+See the [`examples/empire-insider/ReviewAssistant`](./examples/empire-insider/ReviewAssistant.bpmn) diagram
+for such an example.
+
```

Comment:
### Multi-tenancy support

* [Camunda 7 support multi-tenancy identifiers](https://docs.camunda.org/manual/latest/user-guide/process-engine/multi-tenancy/) allowing single process engine to support multiple independent customers. Tenant id could be set in Camunda Modeler, when deploying models and their resources. With `collective.bpmproxy` installed, it is possible to define set of tenancy identifiers for the site by setting configuration registry key `collective.bpmproxy.tenant_ids`. Each Plone site will allow its users to access only those Camunda resources, which are deployed or related to its tenant ids or no tenant ids at all.

Reply:
Added 👍🏾